Update Function editor section (with Project functions functionality) (#9293)
qgis/QGIS-Documentation: docs/user_manual/introduction/qgis_configuration.rst

  version of QGIS`. You can always open projects created with older version of
  QGIS but once the project is saved, trying to open with older release may fail
  because of features not available in that version.

> .. _load_project_code:

Reviewer: ```suggestion

.. _load_project_code:
```

  version of QGIS`. You can always open projects created with older version of
  QGIS but once the project is saved, trying to open with older release may fail
  because of features not available in that version.

> * :guilabel:`Enable project's embedded Python code` |selectString|. This option
>   was created to handle macros that are written to perform an action on project

Reviewer: Let's add an anchor and remove the "was created"
```suggestion
.. _load_project_code:

* :guilabel:`Enable project's embedded Python code` |selectString|. This option
  handles execution of macros that are written to perform an action on project
```
